Description: Arrange for dh_config to be handled by dist-zilla
 (mostly to generate proper man page from pod doc)
Forwarded: not-needed
Author: Dominique Dumont <domi.dumont@free.fr>
Reviewed-by: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2010-11-12
--- a/dist.ini
+++ b/dist.ini
@@ -30,6 +30,10 @@
 [Git::Tag]
 [Git::Push]
 
+[GatherFile]
+filename = debian/dh/dh_cme_upgrade
+prefix = bin
+
 [MetaJSON]
 
 [AutoPrereqs]
